### ISSUE TITLE
Interacting with Tyk GitHub repositories is not permission to add someone to Tyk commercial newsletter lists

### ISSUE THREAD

**Issue by mjpieters** [TykTechnologies/tyk#3625]
I'm filing this as a bug report for this project because this is the source of the problem I am reporting. My GitHub email address has been added to a Tyk newsletter list, shortly after having interacted with this repository, as well as having interacted with another TykTechnologies repo.

**THIS IS NOT COOL**.

I did not give permission to be added to mailinglists by reporting on issues with Tyk.

I interacted with Tyk repositories exactly two times:

- To report an issue with the Tyk gateway helm charts: TykTechnologies/tyk-helm-chart#139
- To make a feature request related to the circumstances that lead to the Helm issue: #3609 

Since my email address attached to these reports is _unique to GitHub_, I know that I ended up on a Hubspot.com list, sporting Tyk commercial news items, because of those two interactions. Volunteers helping you improve your OSS product may well be potential customers, but **volunteering our time to help you is not the same as giving you permission to add us to your mailinglist**.

And while I can simply unsubscribe, as well as reporting this infraction as abusive to hubspot.com, I hereby also give you notice that this practice needs to end now. It impacts my willingness to report issues again in future should I find any, and will do so for others. It greatly undermines the trust that working with the OSS community requires.

**Comment by buger** [comment]
Hi! Thank you raising this issue! It is definitely not expected, and I agree with your point. We will address it ASAP, and if we will find that OSS emails was used in commercial newsletters, we will remove them. I can assure you it was not a strategic decision, most likely just someone thought it can be a good idea...(not).

**Comment by mjpieters** [comment]
@buger thanks for the very prompt follow-up. Do let me know if there is progress on this!

**Comment by buger** [comment]
We found the source of the issue. 

In order to quickly react to GH issues, we decided to integrate GitHub into our standard customer support system (Zendesk). And Zendesk by itself have integration which adds emails of users to Hubspot (to see unified flows across multiple marketing channels). 

And this weird automation, e.g. Github -> Zendesk -> Hubspot, had an issue with setting opt-in marketing email flag. 
We have found 96 affected emails and removed them from our email list. We're still working on a permanent fix, but should be finished in the next few days.
